app/Transformers/AudioTransformer.php

```diff
@@ -57,8 +57,9 @@ public function transformForV2($audio)
              */
             case 'v2_audio_timestamps':
                 return [
-                    'verse_id'    => $audio->verse_start,
-                    'verse_start' => $audio->timestamp
+                    'verse_id'    => (int) $audio->verse_sequence,
```

Comment:
do we need the int cast? I want to be careful not to change the types returned by the API. verse_sequence is already an int, right?